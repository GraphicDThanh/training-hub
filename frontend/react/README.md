# React Redux Training
[Plan doc](https://docs.google.com/document/d/1lYywN0RIhrwsyFZ8uPZuaN5QM78mYuXFD608vS8YEbM/edit?pli=1#)
## ES6
## React Basic
### Practice 1 - examples React
## React Advanced
### Practice 2
## Redux Basic
## Redux Advanced
## Redux Recipes
