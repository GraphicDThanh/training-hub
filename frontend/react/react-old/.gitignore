node_modules
out
dist
.DS_Store
.git
.docpad.db
vendor/
.vagrant
bower-components/
bower_components/
npm-debug.log
.gitignore~
vagrant/Vagrantfile~
build/
