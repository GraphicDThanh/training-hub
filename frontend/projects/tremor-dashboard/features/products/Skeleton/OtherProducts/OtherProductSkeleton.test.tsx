import { render } from "@testing-library/react";

// Components
import OtherProductSkeleton from "./OtherProductSkeleton";

describe("OtherProductSkeleton component", () => {
  it("should match snapshot", () => {
    const { container } = render(<OtherProductSkeleton />);

    expect(container).toMatchSnapshot();
  });
});
