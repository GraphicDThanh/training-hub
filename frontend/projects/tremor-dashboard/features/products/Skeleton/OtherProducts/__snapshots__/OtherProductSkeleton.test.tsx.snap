// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtherProductSkeleton component should match snapshot 1`] = `
<div>
  <div
    class="w-full"
  >
    <b
      class="text-tremor-primary font-semibold dark:text-white"
    >
      Other Products
    </b>
    <div
      class="w-full h-96 mt-5 rounded-lg bg-gray-200 dark:bg-gray-700 animate-pulse"
    />
  </div>
</div>
`;
